content/docs/authorization/deployment/rpm/_index.md

```diff
@@ -72,7 +72,7 @@ A Storage Administrator can execute the shell script, install_karavi_auth.sh as
         "host": ":8080"
       },
       "zipkin": {
-        "collectoruri": "http://DNS-hostname:9411/api/v2/spans",
+        "collectoruri": "http://zipkin-addr:9411/api/v2/spans",
```

Comment:
Does this change also need to be reflected [here](https://github.com/dell/csm-docs/blob/c7350952617f9d00f532ffe1805f24f60b9f6bc2/content/docs/deployment/csmoperator/modules/authorization.md#install-csm-authorization-proxy-server)?

Reply:
Yes, updated. The zipkin address is not the proxy host.